java/code/src/com/suse/manager/webui/controllers/StatesAPI.java

```diff
@@ -667,8 +667,12 @@ public static String showHighstate(Request request, Response response) {
                 .map(minion -> {
                     final String minionId = minion.getMinionId();
                     try {
+                        MinionList minions = new MinionList(minionId);
+                        // bsc#1152673 - sync to minion before showing highstate
```

Comment:
Is it a common practice to keep the bsc#number in comments? We already have it in the commit message, and in Uyuni upstream it will not be reachable by external non-SUSE contributors. I'd remove the number, leaving the comment instead to explain the concept behind.

Reply:
In SUSE it is, but you can find it in the commit message. I will remove it.